### PR TITLE
docs: add clarity for deleting cookies

### PR DESCRIPTION
**Pull request by jarrettmeyer**
Added additional methods for deleting a cookie

### CODE REVIEW

File: docs/02-app/02-api-reference/04-functions/cookies.mdx [hunk 1]
```diff
@@ -85,26 +85,64 @@ async function create(data) {
...
+```js filename="app/actions.js"
+'use server'
+
+import { cookies } from 'next/headers'
+
+async function create(data) {
+  const oneDay = 24 * 60 * 60 * 1000
+  cookies().set('name', 'value', { expires: Date.now() - oneDay })
+}
+```
```
Comment (delbaoliveira) [inline docs/02-app/02-api-reference/04-functions/cookies.mdx:145]:
```suggestion
```

> **Good to know**: You can only delete cookies that belong to the same domain from which `.set()` is called. Additionally, the code must be executed on the same protocol (HTTP or HTTPS) as the cookie you want to delete.
```

File: docs/02-app/02-api-reference/04-functions/cookies.mdx [hunk 2]
```diff
@@ -85,26 +85,64 @@ async function create(data) {
...
```
Comment (delbaoliveira) [inline docs/02-app/02-api-reference/04-functions/cookies.mdx:146]:
```suggestion

> **Good to know**: You can only delete cookies that belong to the same domain from which `.set()` is called. Additionally, the code must be executed on the same protocol (HTTP or HTTPS) as the cookie you want to delete.
```
```

File: docs/02-app/02-api-reference/04-functions/cookies.mdx [hunk 3]
```diff
@@ -85,26 +85,64 @@ async function create(data) {
...
-    value: '',
-    expires: new Date('2016-10-05'),
-    path: '/', // For all paths
-  })
+  cookies().delete('name');
 }
 ```
 
-You can only set cookies that belong to the same domain from which `.set()` is called. Additionally, the code must be executed on the same protocol (HTTP or HTTPS) as the cookie you want to update.
+### `cookies().set(name, '')`
```
Comment (delbaoliveira) [inline docs/02-app/02-api-reference/04-functions/cookies.mdx:106]:
```suggestion
### `cookies().set(name, '')`

Alternatively, you can set a new cookie with the same name and an empty value.```

**Review by delbaoliveira — DISMISSED** [review]
Nice, thank you @jarrettmeyer!

File: docs/02-app/02-api-reference/04-functions/cookies.mdx [hunk 4]
```diff
@@ -85,27 +85,68 @@ async function create(data) {
...
+
+import { cookies } from 'next/headers'
+
+async function create(data) {
+  const oneDay = 24 * 60 * 60 * 1000
+  cookies().set('name', 'value', { expires: Date.now() - oneDay })
+}
+```
 
+> **Good to know**: You can only delete cookies that belong to the same domain from which `.set()` is called. Additionally, the code must be executed on the same protocol (HTTP or HTTPS) as the cookie you want to delete.
```
Comment (leerob) [inline docs/02-app/02-api-reference/04-functions/cookies.mdx:149]:
```suggestion
> **Good to know**: You can only delete cookies that belong to the same domain from which `.set()` is called. Additionally, the code must be executed on the same protocol (HTTP or HTTPS) as the cookie you want to delete.

```

**Review by leerob — APPROVED** [review]

**Review by leerob — DISMISSED** [review]

**Review by ijjk — DISMISSED** [review]

**Review by ijjk — APPROVED** [review]

**Pull request merged**